src/archivematicaCommon/lib/archivesspace/client.py

```diff
@@ -0,0 +1,424 @@
+import json
+import os
+import requests
+from urlparse import urlparse
+
+
+class ConnectionError(Exception):
+    pass
+
+
+class AuthenticationError(Exception):
+    pass
+
+
+class CommunicationError(Exception):
```

Comment:
Should these inherit from `ArchivesSpaceError`? So it's easier to catch anything generated by this module?